libraries/ESP8266WiFi/src/ESP8266WiFiType.h

```diff
@@ -58,9 +59,10 @@ typedef enum WiFiEvent
     WIFI_EVENT_SOFTAPMODE_STACONNECTED,
     WIFI_EVENT_SOFTAPMODE_STADISCONNECTED,
     WIFI_EVENT_SOFTAPMODE_PROBEREQRECVED,
+    WIFI_EVENT_MODE_CHANGE,

```

Comment:
Maybe swap these to keep the softapmode together

Reply:
The order needs to be the same as [SDK defines](https://github.com/esp8266/Arduino/blob/0399bb1fee4b95bfb8f9007f561e4f7060eedcdb/tools/sdk/include/user_interface.h#L466)'s